src/pages/Polling.js

```diff
@@ -340,11 +340,13 @@ class Polling extends React.Component {
       this.state.activeAccount !== prevProps.activeAccount ||
       (this.props.poll && prevProps.poll === undefined)
     ) {
-      await this.updateVotedPollOption();
+      this.props.pollDataInit(this.props.poll);
```

Comment:
initiate poll data fetch from the polling details page based on the `pollId` of that page